client/src/backend/components/list/EntitiesList/Entity/ProjectExportationRow.js

```diff
@@ -0,0 +1,32 @@
+import React, { PureComponent } from "react";
+import PropTypes from "prop-types";
+import FormattedDate from "global/components/FormattedDate";
+import EntityRow from "./Row";
+
+export default class ProjectExportationRow extends PureComponent {
+  static propTypes = {
+    entity: PropTypes.object.isRequired
+  };
+
+  render() {
+    const {
+      entity: {
+        attributes: { createdAt, exportedAt, currentState },
+        relationships: {
+          exportTarget: {
+            attributes: { name: exportTargetName }
```

Comment:
I haven't seen a lot of advanced es6 destructuring in this project, but I think that in this case, it is a way of assigning variables from this prop that makes it a little easier to read. Any thoughts on this? I want to make sure I'm not breaking any conventions here.